lib/client-assets.php

```diff
@@ -146,24 +146,45 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'i18n/build/index.js' ),
 		true
 	);
+	wp_register_script(
```

Comment:
Should it be registered before it is referenced as a dependency or it doesn't matter at all?

Reply:
>Should it be registered before it is referenced as a dependency or it doesn't matter at all?

I don't know that it matters but I'll update in a rebase.